src/config.js

```diff
@@ -254,7 +254,7 @@ export const config = configOrder({
 		type: 'object',
 		properties: {
 			webgl: {
-				title: 'WebGL Renderer',
+				title: 'WebGL Renderer ⚠️EXPERIMENTAL⚠️',
```

Comment:
```suggestion
				title: '⚠️ Experimental WebGL Renderer',
```
:+1:

Reply:
Again is the caution on Experimental or WebGL. Keep in mind that some people (probably most people) won't know what WebGL is.

Reply:
I also want it to convey that the renderer is experimental not WebGL.

Reply:
>I also want it to convey that the renderer is experimental not WebGL.

Then neither is good for that.

The description is the way to go for that not the title. For that we need.

`description: '⚠️ Enable the WebGL-based renderer using the experimental xterm.js [WebGL addon](https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-webgl) ',

Now that conveys the message clearly

Reply:
With the caution symbol at the beginning I don't know if you are saying "Hey, you might want to enable me!" or "Hey, you might not want to enable me!" because it is not clear that the reason for the symbol is because it is experimental.

Reply:
I guess that would work without the symbol, but I'm not sure people would see it.

![image](https://user-images.githubusercontent.com/97994/91228784-6fc4e100-e6ee-11ea-92a9-65e6fa8d181f.png)

Reply:
bold the word **experimental**

Reply:
![image](https://user-images.githubusercontent.com/97994/91480096-11bc0900-e868-11ea-81b9-f84d4b735a8c.png)

Reply:
=) All capitals as a way to emphasize a specific word in this way?

Reply:
It makes it really easy to see 😁

Reply:
https://en.wikipedia.org/wiki/All_caps says

>Studies have been conducted on the readability and legibility of all caps text. Scientific testing from the 20th century onward has generally indicated that **all caps text is less legible and readable than lower-case text**.[2][3] In addition, **switching to all caps may make text appear hectoring and obnoxious for cultural reasons**, since all-capitals is often used in transcribed speech to indicate that the speaker is shouting.[4] All-caps text is common in comic books, as well as on older teleprinter and radio transmission systems, which often do not indicate letter case at all.[5][6]

That said, the real problem(s) as I see is that both the option title and description font color in this particular Atom theme (I also use). If the font color would be brighter there would be better contrast, (as is its a lower level of contrast) thus making the words more readable by an increased font brightness..

Also most users dont read anything anyway, so this leads me to question how serious I am about this single instance, and tbh that last observation about general users, makes me not care about this instance, even if some part of me is somewhat OCD about it =)

On another note;

I'll try to submit a PR to address this color contrast in Atom's applicable themes but am dreading the undertaking cause I'll be fixing issues other than that, OCD ugh, and some Atom themes are somewhat broken and the 2 minute job will turn into hours.

Reply:
![image](https://user-images.githubusercontent.com/97994/91566963-04575b00-e90a-11ea-99f7-9c40b608e224.png)

Reply:
or what about adding a short description at the end.

![image](https://user-images.githubusercontent.com/97994/91567224-69ab4c00-e90a-11ea-9399-ddd4792a1a06.png)

Reply:
You are creating two sentences where one was and making the **pertinent** word all the way to the end of first sentence.

Reply:
Why is it bad to have more than one sentence? And when people skim something they tend to remember the beginning and the end more than the middle.

Reply:
This seems like a good way to let people know why they might want to enable and why they might not.

Reply:
As is now is fine =)